sentinel-2/kndvi/README.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: kNDVI with Cloud Masking
+parent: Sentinel-2
+title: Contribute
+grand_parent: Sentinel
+layout: script
+permalink: /sentinel-2/kndvi/
+nav_exclude: true
+nav_order: 99
+has_children: false
+permalink: /contribute/
+---
```

Comment:
It would be a good practice to have the raw values script, the visualisation script, and the copernicus browser script  which enables statistical feature as the Sentinel-2 [NDVI example](https://raw.githubusercontent.com/sentinel-hub/custom-scripts/main/sentinel-2/ndvi/README.md).

Reply:
do we prefer to have these in separate scripts instead of one script providing all the outputs, as originally written?

Reply:
The idea is to have all types of scripts displayed, i.e., a raw value script which only returns the index which is kndvi in your case, a visualisation script which only returns the visualisation which is the default part of your script, and the copernicus browser script which enables statistical features which is exactly the script you added.

The reason is that users often copy and paste the scripts here to either their process api request or eo/copernicus browser. Originally we only put the visualisation script, but we got a lot of questions from the users about the script not returning the actual index and not returning correct stats on the browser. We decided to display all types of scripts at the end, so we can easily point users to the right tab like this [page](https://custom-scripts.sentinel-hub.com/sentinel-2/ndvi/).

Reply:
I would still prefer to have all three outputs in one script as prepared when coding together

Reply:
I think we have a misunderstanding here. We will still have the script with all outputs, which is the `eob.js` (see the [example](https://github.com/sentinel-hub/custom-scripts/blob/main/sentinel-2/ndvi/eob.js#L47)). The eob.js is the script designed to be used in EO Browser and Copernicus Browser with the statistical feature enabled.

The idea to **also** include scripts.js (visualisation) and raw.js (raw values) is that users do not only use the scripts provided in the custom scripts repository in the browsers but also use them in the requests and the requests builder. In this case they are interested in getting the visualisation  and/or the raw values (in your case the actual kNDVI). It would be really convenient for them to copy the visualisation or raw value scripts to their request and/or the requests builder to get what they want. It is also convenient for us to point the user to the right script if we have all types of scripts available on the website.